content/developer/reference/frontend/odoo_editor.rst

```diff
@@ -0,0 +1,120 @@
+
+.. highlight:: javascript
+
+.. default-domain:: js
+
+===========
+Odoo Editor
+===========
+
+Odoo Editor is Odoo's own rich text editor. It has replaced the Summernote
+library since Odoo 15.0. It can be found at
+*web_editor/static/src/js/editor/odoo-editor*.
```

Comment:
```suggestion
library since Odoo 15.0. Its sources can be found in the `odoo-editor directory
<{GITHUB_PATH}/addons/web_editor/static/src/js/editor/odoo-editor>`_.
```

Reply:
Why do you put `odoo-editor directory` in code along with the path? Also I'm guessing the trailing `_` is not intended. I had put the path between `*` because that's what I had seen in other pages of the doc. Is that not the guideline then?